bookserver/routers/books.py

```diff
@@ -103,11 +104,11 @@ async def serve_page(
     #   activity_info
     #   settings.google_ga
     await create_useinfo_entry(
```

Comment:
I'm not following this, and it appears to be a step back from using a well defined schema to a dictionary.  Can you give me some background on your thinking and the rationale for doing this?

Reply:
I'd like a chance to talk with you about this -- when are you free?